modules/busco/main.nf

```diff
@@ -24,15 +25,26 @@ process BUSCO {
     """
     # Ensure the input is uncompressed
     gzip -cdf $fasta > __UNCOMPRESSED_FASTA_FILE__
+    
+    # Nextflow changes the container --entrypoint to /bin/bash (container default entrypoint: /usr/local/env-execute)
+    # Check for container variable initialisation script and source it.
+    if [ -f "/usr/local/env-activate.sh" ]; then
+        # . "/usr/local/env-activate.sh"  # Errors out because of various unbound variables
+        export PATH='/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
+        export CONDA_PREFIX='/usr/local'
+        export CONDA_SHLVL='1'
+        export CONDA_DEFAULT_ENV='/usr/local'
+        export CONDA_PROMPT_MODIFIER=''
+        . "/usr/local/etc/conda/activate.d/activate-r-base.sh"
+        . "/usr/local/etc/conda/activate.d/augustus.sh"
+        . "/usr/local/etc/conda/activate.d/openjdk_activate.sh"
+    fi
+    
     # Copy the image's AUGUSTUS config directory if it was not provided to the module
     [ ! -e augustus_config ] && cp -a /usr/local/config augustus_config
```

Comment:
```suggestion
    [ ! -e $augustus_config ] && cp -a /usr/local/config $augustus_config
```
This is what @muffato was referring to in their comment. The variable in your input declaration is not being used in your script.

Reply:
OK. More feedback. So technically this line is not doing anything at the moment.
Is there a need for this folder to be provided as input?
```
    # If the augustus config directory is not writable, then copy to writeable area
    if [ ! -w "\${AUGUSTUS_CONFIG_PATH}" ]; then
        # Create writable tmp directory for augustus
        AUG_CONF_DIR=\$( mktemp -d -p \$PWD )
        cp -r \$AUGUSTUS_CONFIG_PATH/* \$AUG_CONF_DIR
        export AUGUSTUS_CONFIG_PATH=\$AUG_CONF_DIR
        echo "New AUGUSTUS_CONFIG_PATH=\${AUGUSTUS_CONFIG_PATH}"
    fi
```
The folder just needs to be a writable temp dir to do augustus stuff as far as I understand. One can then change the busco line below as the config path assignment is then not needed.